src/posts/2021-08-14-reviewing-a-design-for-accesibility.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Reviewing a design for accessibility
+description: "Reviewing designs for accessibility is a great way to help designers think more accessibly and to help developers implement more accessibly. We can use short checklists to help us make sure we haven't missed any big things."
+category: Quick tip
+author: Steve Barnett
+date: 2021-08-14
+tags:
+  - quicktip
+---
+
+I'm a big believer in [shifting left](https://www.deque.com/shift-left/): considering accessibility as early in the process as we can. One way we can do this is by reviewing designs for accessibility. This helps us catch anything we've missed before we start developing it.
+
+## What to do: annotate
+
+By adding annotations about the semantics and accessibility of elements on a page we can remove barriers to access. The best way to do annotations might be sticky notes on a printout or comments on a Figma file, or something else. The important thing is that we want to be clear and explicit.
+
+Here are a handful of things that I've found give good value / effort balance when reviewing a design for accessibility. 
+
+## Things to check
+
+- **Page title**
+    - Is the page title unique?
+    - Does it describe the page content?
```

Comment:
```suggestion
    - Does it concisely describe the page content?
```

I am suggesting this if only because my bank took this too literally and dumped the entire intro paragraph into their page title 😓